frontend/src/screens/channels/IncreaseOutgoingCapacity.tsx

```diff
@@ -429,6 +443,80 @@ function NewChannelInternal({
           </ExternalLinkButton>
         </div>
       </div>
+
+      {/* Confirmation Modal */}
+      <Dialog open={showConfirmModal} onOpenChange={setShowConfirmModal}>
+        <DialogContent className="sm:max-w-md">
+          <DialogHeader>
+            <DialogTitle>Confirm Channel Opening</DialogTitle>
+            <DialogDescription>
+              Are you sure you want to open a Lightning channel with the
+              following details?
+            </DialogDescription>
+          </DialogHeader>
+
+          <div className="space-y-4">
+            <div className="grid grid-cols-2 gap-4 text-sm">
+              <div>
+                <div className="font-medium text-muted-foreground">Peer</div>
+                <div>{selectedPeer?.name || "Custom"}</div>
+              </div>
+              <div>
+                <div className="font-medium text-muted-foreground">Amount</div>
+                <div>
+                  {formatAmount(parseInt(order.amount || "0") * 1000)} sats
```

Comment:
Actually here I think we shouldn't use the `formatAmount` function and just display the full amount of sats using `new Intl.NumberFormat().format(parseInt(order.amount || "0"))`